skimage/registration/_phase_cross_correlation.py

```diff
@@ -111,8 +113,74 @@ def _compute_error(cross_correlation_max, src_amp, target_amp):
     return np.sqrt(np.abs(error))
 
 
+def _disambiguate_shift(reference_image, moving_image, shifts):
+    """Determine the correct real-space shift based on periodic shift.
+
+    When determining a translation shift from phase cross-correlation in
+    Fourier space, the shift is only correct to within a period of the image
+    size along each axis, resulting in $2^n$ possible shifts, where $n$ is the
+    number of dimensions of the image. This function checks the
+    cross-correlation in real space for each of those shifts, and returns the
+    one with the highest cross-correlation.
+
+    The strategy we use is to perform the shift on the moving image *using the
+    'grid-wrap' mode* in `scipy.ndimage`. The original image borders then
+    define $2^n$ quadrants, which we cross-correlate with the reference image
+    in turn using slicing. The entire operation is thus $O(2^n + m)$, where
+    $m$ is the number of pixels in the image (and typically dominates).
+
+    Parameters
+    ----------
+    reference_image : numpy array
+        The reference (non-moving) image.
+    moving_image : numpy array
+        The moving image: applying the shift to this image overlays it on the
+        reference image. Must be the same shape as the reference image.
+    shifts : tuple of float
```

Comment:
```suggestion
    shift : tuple of float
```
I realize there is a mix of singular and plural even in the private function: I suggest using the singular (as in the description part of the docstring), so that by 'shift' we mean the tuple of shifts.